src/tlo/methods/healthsystem.py

```diff
@@ -1282,10 +1282,16 @@ def schedule_hsi_event(
         """
         # If there is no specified tclose time then set this to a week after topen.
         # This should be a boolean, not int! Still struggling to get a boolean variable from resource file
-        if self.tclose_overwrite == 1:
-            tclose = topen + pd.to_timedelta(self.tclose_days_offset_overwrite, unit='D')
-        elif tclose is None:
-            tclose = topen + DateOffset(days=7)
+
+        # Clinical time-constraints are embedded in tclose for these modules, do not overwrite their tclose
+        if hsi_event.module == ('CareOfWomenDuringPregnancy' or 'Labour' or 'PostnatalSupervisor' or 'NewbornOutcomes'):
+            if tclose is None:
+                tclose = topen + DateOffset(days=7)
```

Comment:
here, and in line 1294, we have the defaulting to  7 days for `tclose`. I guess it happens somewhere else too. I think we should factorize this so that we have one declaration of the default value for an unspecified `tclose`.